ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/domain/Rejection.scala

```diff
@@ -64,6 +64,14 @@ private[sandbox] object Rejection {
       .rpcStatus(completionInfo.submissionId)
   }
 
+  final case class OffsetDeduplicationPeriodUnsupported(completionInfo: CompletionInfo)(implicit
+      contextualizedErrorLogger: ContextualizedErrorLogger
+  ) extends Rejection {
+    override def toStatus: Status = LedgerApiErrors.UnsupportedOperation
+      .Reject("Command deduplication period of type offset is not supported.")
```

Comment:
```suggestion
      .Reject("command deduplication with periods specified using offsets")
```

which results in the message "The request exercised an unsupported operation: command deduplication with periods specified using offsets"